pages/docs/reference/whatsnew14.md

```diff
@@ -604,7 +604,7 @@ When using the command-line compiler, add the compiler option `-Xuse-ir`.
 > Considering this, we don't recommend that library authors switch to the new back-end in production.
 {:.note}
 
-You can also opt into using the new JS IR back-end (ADD LINK).
+You can also opt into using the new [JS IR back-end](#JS-IR-compiler-backend).
```

Comment:
```suggestion
You can also opt into using the new [JS IR back-end](#new-JS-IR-back-end).
```

Reply:
Thanks. After merging a current version of the 1.4 docs branch, this line has been removed. I've made sure that the other anchors are assigned correctly.